case-server/pom.xml

```diff
@@ -125,6 +130,25 @@
             <artifactId>log4j-over-slf4j</artifactId>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>com.powsybl</groupId>
+            <artifactId>powsybl-triple-store-impl-rdf4j</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>
```

Comment:
You don't have to specify rdf4j dependencies, it is transitively pulled by powsybl-triple-store-impl-rdf4j.

Reply:
It is to fix the error when we get a case with xiidm format :
curl -X GET "http://localhost:5000/v1/cases/c23b830e-92aa-4804-b911-614dc541349b?xiidm=true" -H "accept: */*"
com.powsybl.commons.PowsyblException: No implementation available for triple store rdf4j